docs/content/posts/2024-07-15-release-1.20.0.md

```diff
@@ -0,0 +1,510 @@
+- `table.exec.range-sort.enabled`
+- `table.optimizer.rows-per-local-agg`
+- `table.optimizer.join.null-filter-threshold`
+- `table.optimizer.semi-anti-join.build-distinct.ndv-ratio`
+- `table.optimizer.shuffle-by-partial-key-enabled`
+- `table.optimizer.smj.remove-sort-enabled`
+- `table.optimizer.cnf-nodes-limit`
+
+These options were introduced for the now-obsolete `FilterableTableSource` interface:
+- `table.optimizer.source.aggregate-pushdown-enabled`
+- `table.optimizer.source.predicate-pushdown-enabled`
+
+**More Information**
+* [FLINK-35461](https://issues.apache.org/jira/browse/FLINK-35461)
+* [FLINK-35473](https://issues.apache.org/jira/browse/FLINK-35473)
+
+## New and Updated Configuration Options
+
+### SQL Client Option
+`sql-client.display.max-column-width` has been replaced with `table.display.max-column-width`.
+
+### Batch Execution Options
+The following options have been moved from `org.apache.flink.table.planner.codegen.agg.batch.HashAggCodeGenerator`
+to `org.apache.flink.table.api.config` and promoted to `@PublicEvolving`.
+
+- `table.exec.local-hash-agg.adaptive.enabled`
+- `table.exec.local-hash-agg.adaptive.sampling-threshold`
+- `table.exec.local-hash-agg.adaptive.distinct-value-rate-threshold`
+
+### Lookup Hint Options
+The following options have been moved from `org.apache.flink.table.planner.hint.LookupJoinHintOptions`
+to `org.apache.flink.table.api.config.LookupJoinHintOptions` and promoted to `@PublicEvolving`.
+
+- `table`
+- `async`
+- `output-mode`
+- `capacity`
+- `timeout`
+- `retry-predicate`
+- `retry-strategy`
+- `fixed-delay`
+- `max-attempts`
+
+### Optimizer Options
+The following options have been moved from `org.apache.flink.table.planner.plan.optimize.RelNodeBlock`
+to `org.apache.flink.table.api.config.OptimizerConfigOptions` and promoted to `@PublicEvolving`
+
+- `table.optimizer.union-all-as-breakpoint-enabled`
+- `table.optimizer.reuse-optimize-block-with-digest-enabled`
+
+### Aggregate Optimizer Option
+`table.optimizer.incremental-agg-enabled` has been moved from `org.apache.flink.table.planner.plan.rules.physical.stream.IncrementalAggregateRule`
+to `org.apache.flink.table.api.config.OptimizerConfigOptions` and promoted to `@PublicEvolving`
+
+**More Information**
+* [FLINK-35461](https://issues.apache.org/jira/browse/FLINK-35461)
+* [FLINK-35473](https://issues.apache.org/jira/browse/FLINK-35473)
+
+# Upgrade Notes
+
+The Flink community tries to ensure that upgrades are as seamless as possible.
+However, certain changes may require users to make adjustments to certain parts
+of the program when upgrading to version 1.20. Please refer to the
+[release notes](https://nightlies.apache.org/flink/flink-docs-release-1.20/release-notes/flink-1.20/)
+for a comprehensive list of adjustments to make and issues to check during the
+upgrading process.
+
+# List of Contributors
+
+The Apache Flink community would like to express gratitude to all the
+contributors who made this release possible:
```

Comment:
Will this be rendered as a list (a newline after each name?) In the past, this was just one block of names: https://flink.apache.org/2024/03/18/announcing-the-release-of-apache-flink-1.19/#list-of-contributors